src/client.ts

```diff
@@ -181,41 +188,58 @@ export default class Client {
   }
 
   private connect() {
-    this.client = new W3CWebSocket(this.url, GRAPHQL_SUBSCRIPTIONS);
+    this.client = new WebSocket(this.url, GRAPHQL_SUBSCRIPTIONS);
 
     this.client.onopen = () => {
       this.reconnecting = false;
       this.backoff.reset();
       Object.keys(this.reconnectSubscriptions).forEach((key) => {
         const { options, handler } = this.reconnectSubscriptions[key];
         this.subscribe(options, handler);
-      })
+      });
       this.unsentMessagesQueue.forEach((message) => {
         this.client.send(JSON.stringify(message));
       });
       this.unsentMessagesQueue = [];
+
+      let message = Object.assign({type: INIT, payload: this.connectionParams});
```

Comment:
Can be const, or just inlined into next line.  Also single-argument Object.assign seems unnecessary.